Author: Matthijs Kooijman <matthijs@stdin.nl>
Description: Modify the right-to-left language error message
 Rather than suggesting the user to recompile with ICU support (which
 won't work on Debian, since the required ParagraphLayout API is not
 enabled in ICU), point them at README.Debian which explains this.
Forwarded: not-needed
--- a/src/strings.cpp
+++ b/src/strings.cpp
@@ -2181,7 +2181,7 @@ void CheckForMissingGlyphs(bool base_fon
 	 * the colour marker.
 	 */
 	if (_current_text_dir != TD_LTR) {
-		static char *err_str = stredup("XXXThis version of OpenTTD does not support right-to-left languages. Recompile with icu enabled.");
+		static char *err_str = stredup("XXXThis version of OpenTTD does not support right-to-left languages. See README.Debian for details.");
 		Utf8Encode(err_str, SCC_YELLOW);
 		SetDParamStr(0, err_str);
 		ShowErrorMessage(STR_JUST_RAW_STRING, INVALID_STRING_ID, WL_ERROR);
